coalib/bears/Bear.py

```diff
@@ -188,6 +188,8 @@ class Bear(Printer, LogPrinterMixin, metaclass=bearclass):
     SEE_MORE = ''
     BEAR_DEPS = set()
     USE_RAW_FILES = False
+    ACCEPTABLE_VALUES_FOR_SETTINGS = []
```

Comment:
this information should not be stored in the bear like this.  

it needs to be per setting, and using type annotations.  This was already discussed in the cEP.

that allows re-use of the type annotation.

Reply:
can't find an elegant way to use type annotations for value checks https://github.com/coala/coala-bears/issues/2532

Reply:
if what you mean by this:
>it needs to be per setting,

is
>Alternate approach could have been the bear writer populating a field else where and calling the decorator with those fields that quickstart can utilise directly but it just felt inelegant and contradicting the process of making bear development easier and easier by making changes at two different places to achieve a single task

Then I can make the necessary changes

Reply:
>this information should not be stored in the bear like this.

where exactly should it be stored then?

Reply:
> where exactly do you want it to be stored?

inside the annotation!

An annotation can be a class, with any amount of logic and data inside it.

Reply:
but we can't use that unless coala is used with the `mypy` package loaded

Reply:
https://github.com/coala/coala/pull/5523#discussion_r195269799

Reply:
have you seen my comment at https://github.com/coala/coala-bears/issues/2532 ?

Reply:
yes